topic/chatbot/dis2025/workshop/README.md

```
@@ -0,0 +1,68 @@
+# Workshop Title:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)